samples/client/petstore/csharp-netcore/OpenAPIClient-httpclient/src/Org.OpenAPITools/Client/ApiClient.cs

```diff
@@ -337,10 +337,13 @@ private ApiResponse<T> ToApiResponse<T>(HttpResponseMessage response, object res
 
             if (response != null)
             {
-                foreach (Cookie cookie in handler.CookieContainer.GetCookies(uri))
-                {
-                   transformed.Cookies.Add(cookie);
+                try {
+                    foreach (Cookie cookie in handler.CookieContainer.GetCookies(uri))
+                    {
+                       transformed.Cookies.Add(cookie);
+                    }
                 }
+                catch (PlatformNotSupportedException) {}
```

Comment:
Shall we log a warning message or use other channels to notify the users/developers? It seems like the exception is now discarded silently.

Reply:
A good question and one I've also asked myself. The issue is that PlatformNotSupported is a very specific exception that I think is only thrown on Blazor WASM right now (as its running in Browser).

If we start logging this now this will clutter up the console log of anyone using the generated code for any api call on that platform. We could introduce a logging level cli switch for generating code that would introduce log messages here and at other places throughout the code. That might be an interesting addition anyway.

Reply:
> If we start logging this now this will clutter up the console log of anyone using the generated code for any api call on that platform

What about we start with this and revise the logging strategy later if users complain about it?

I still prefer over-communication to the users/developers to start with because I'd bad experiences dealing with errors being silent before.

Reply:
Let's go with what you've so far to start with. We can always improve later with another PR.